boards/arm/cxd56xx/drivers/camera/Make.defs

```diff
@@ -0,0 +1,42 @@
+############################################################################
+# boards/arm/cxd56xx/drivers/sensors/Make.defs
+#
+#   Copyright 2018 Sony Semiconductor Solutions Corporation
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in
+#    the documentation and/or other materials provided with the
+#    distribution.
+# 3. Neither the name of Sony Semiconductor Solutions Corporation nor
+#    the names of its contributors may be used to endorse or promote
+#    products derived from this software without specific prior written
+#    permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+# OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+# AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+############################################################################
```

Comment:
I know that you are really only moving a file, but the requirement is that all new files must have Apache 2.0 headers or they cannot come into the repostiory.  Please replace this header with one like that in sched/sched/sched_getcpu.c -- Note that there is not copyright or author information in the header.

Reply:
The same applies to ixx012.c which Github in all of its wisdom will not let me see.

Reply:
I am seeing this error in the automated build runs:

    ====================================================================================
    Configuration/Tool: spresense/audio,CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
    ------------------------------------------------------------------------------------
      Cleaning...
      Configuring...
      Copy files
      Select CONFIG_HOST_LINUX=y
      Refreshing...
      Disabling CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
      Enabling CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
      Refreshing...
      Building NuttX...
    ------------------------------------------------------------------------------------
    platform/camera/Make.defs:1: *** missing separator.  Stop.
    platform/camera/Make.defs:1: *** missing separator.  Stop.
    make: *** [drivers/libdrivers.a] Error 2
    make: *** Waiting for unfinished jobs....
    make: *** wait: No child processes.  Stop.
    ##[error]Process completed with exit code 2.

Reply:
Fixed by PR #468